-2_values.ipynb

```diff
@@ -109639,7 +109542,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.12.2"
+   "version": "3.11.6"
```

Comment:
I was working on both locally and through BU SCC. I'm guessing there's some inconsistency in the version because of the way we set up our Conda environment. Shouldn't cause any problems in these notebooks though.